Makefile

```diff
@@ -121,10 +121,8 @@ test-coverage: fmt lint test-coverage-tools | $(BASE) ; $(info  Running coverage
 	$Q $(GOCOV) convert $(COVERAGE_PROFILE) | $(GOCOVXML) > $(COVERAGE_XML)
 
 .PHONY: lint
-lint: | $(BASE) $(GOLINT) ; $(info  Running golint...) @ ## Run golint on all source files
-	$Q cd $(BASE) && ret=0 && for pkg in $(PKGS); do \
-		test -z "$$($(GOLINT) $$pkg | tee /dev/stderr)" || ret=1 ; \
-	 done ; exit $$ret
+lint: | $(BASE) $(GOLANGCILINT) ; $(info  Running golangci-lint...) @ ## Run golint on all source files
+	$Q $(GOLANGCILINT) run --enable=revive,bodyclose,gosec,whitespace ./...
```

Comment:
could we have .golangci.yml in project with linter config ? (similar to sriov dp)

then we can drop configuration here and in workflow